data/sql/updates/pending_db_world/rev_1588337316614118000.sql

```diff
@@ -0,0 +1,64 @@
+UPDATE `creature_template` SET `mindmg` = 116, `maxdmg` = 156, `DamageModifier` = 1.03 WHERE `entry` = 6427;
+UPDATE `creature_template` SET `mindmg` = 146, `maxdmg` = 189, `DamageModifier` = 1.03 WHERE `entry` = 6493;
+UPDATE `creature_template` SET `mindmg` = 116, `maxdmg` = 156, `DamageModifier` = 1.03 WHERE `entry` = 6426;
+UPDATE `creature_template` SET `mindmg` = 112, `maxdmg` = 156, `DamageModifier` = 1.03 WHERE `entry` = 4308;
+UPDATE `creature_template` SET `mindmg` = 103, `maxdmg` = 140, `DamageModifier` = 1.03 WHERE `entry` = 4287;
+UPDATE `creature_template` SET `mindmg` = 103, `maxdmg` = 140, `DamageModifier` = 1.03 WHERE `entry` = 4296;
+UPDATE `creature_template` SET `mindmg` = 124, `maxdmg` = 171, `minrangedmg` = 89, `maxrangedmg` = 136, `DamageModifier` = 1.03 WHERE `entry` = 4288;
+UPDATE `creature_template` SET `mindmg` = 93, `maxdmg` = 125, `DamageModifier` = 1.03 WHERE `entry` = 4304;
+UPDATE `creature_template` SET `mindmg` = 106, `maxdmg` = 140, `DamageModifier` = 1.03 WHERE `entry` = 4299;
+UPDATE `creature_template` SET `mindmg` = 106, `maxdmg` = 140, `DamageModifier` = 1.03 WHERE `entry` = 4291;
+UPDATE `creature_template` SET `mindmg` = 115, `maxdmg` = 154, `DamageModifier` = 1.03 WHERE `entry` = 4540;
+UPDATE `creature_template` SET `mindmg` = 131, `maxdmg` = 177, `DamageModifier` = 1.03 WHERE `entry` = 4286;
+UPDATE `creature_template` SET `mindmg` = 83, `maxdmg` = 115, `DamageModifier` = 1.03 WHERE `entry` = 4297;
+UPDATE `creature_template` SET `mindmg` = 113, `maxdmg` = 152, `DamageModifier` = 1.03 WHERE `entry` = 575;
+UPDATE `creature_template` SET `mindmg` = 83, `maxdmg` = 118, `DamageModifier` = 1.03 WHERE `entry` = 4289;
+UPDATE `creature_template` SET `mindmg` = 108, `maxdmg` = 147, `DamageModifier` = 1.03 WHERE `entry` = 4292;
+UPDATE `creature_template` SET `mindmg` = 184, `maxdmg` = 255, `DamageModifier` = 1.03 WHERE `entry` = 4290;
+UPDATE `creature_template` SET `mindmg` = 115, `maxdmg` = 155, `DamageModifier` = 1.03 WHERE `entry` = 4295;
+UPDATE `creature_template` SET `mindmg` = 143, `maxdmg` = 199, `DamageModifier` = 1.03 WHERE `entry` = 4301;
+UPDATE `creature_template` SET `mindmg` = 30, `maxdmg` = 42, `DamageModifier` = 1.03 WHERE `entry` = 6575;
+UPDATE `creature_template` SET `mindmg` = 90, `maxdmg` = 129, `DamageModifier` = 1.03 WHERE `entry` = 4300;
+UPDATE `creature_template` SET `mindmg` = 87, `maxdmg` = 125, `DamageModifier` = 1.03 WHERE `entry` = 4294;
+UPDATE `creature_template` SET `mindmg` = 121, `maxdmg` = 161, `DamageModifier` = 1.03 WHERE `entry` = 4302;
+UPDATE `creature_template` SET `mindmg` = 121, `maxdmg` = 161, `DamageModifier` = 1.03 WHERE `entry` = 4303;
+
+/* RARE */
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 174, `maxdmg` = 224, `DamageModifier` = 1.02 WHERE `entry` = 6490;
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 174, `maxdmg` = 224, `DamageModifier` = 1.02 WHERE `entry` = 6488;
```

Comment:
Just to make less queries
```suggestion
UPDATE `creature_template` SET `rank` = 2, `mindmg` = 174, `maxdmg` = 224, `DamageModifier` = 1.02 WHERE `entry` IN (6490, 6488);
```